client/tracing/src/block/mod.rs

```diff
@@ -301,12 +310,12 @@ where
 	}
 }
 
-fn event_key_filter(event: &TraceEvent, storage_keys: &str) -> bool {
+fn event_values_filter(event: &TraceEvent, key: &str, values: &str) -> bool {
```

Comment:
I think "key" is an inflated word and we should strive to be more specific. Would `filter_kind` work here?